README.md

```diff
@@ -65,6 +67,12 @@ The exporter collects a number of statistics from the server:
 Request metrics:
 
 ```
+# HELP apache_requests_per_second Requests per second*
+# TYPE apache_requests_per_second counter
```

Comment:
Thanks for the patch!!

Here is my opinion:

We should not have that.

This is in the opposite of Prometheus philosophy.

Just use rate(apache_requests_total[1m]) to have the information.

And it should be a gauge.

Reply:
Thanks for feedback. 
I am totally agree with metrics type suggestions, will fix that.
As for request rate, I would love to use apache_requests_total as a rate. The thing is it does not show total request count. In my case it shows only below metric:
```
http_requests_total{code="200",handler="prometheus",method="get"} 6248
```
Can you please check/confirm apache_requests_total metric shows real total number of requests for apache in your installation?

Reply:
What you want is:
sum(rate(http_requests_total[1m]))

You probably just have return codes 200 at the moment.

Reply:
http_requests_total shows requests only for apache exporter activity. It does not count requests made by real users which are hitting my apache server.

Reply:
let me check

Reply:
From my understanding it shows request count being made to exporter itself.

Reply:
In any case, we can get the same number by applying rate to accesses_total right?

Reply:
I am looking into apache docs, and can't figure what does they mean by "A total number of accesses"? Is it total number of requests? If so, then it perfectly fits.

Reply:
as @dborysenko pointed out, the http_requests_total is referring to the exporter itself.

https://github.com/prometheus/client_golang/blob/535f493f5a875ebde111871209251257ef834ad1/prometheus/http.go#L153

Reply:
sorry indeed, apache_accesses_total

Reply:
I can't find any detailed explanation about total accesses either, but seems to be what we need.
In conclusion, the PR should only include the new cpu usage metric and should be a gauge.

Reply:
Comparing apache_requests_per_second with rate(apache_accesses_total[5m]). So far they look similar. 
Will prepare commit without "request per second" and others, just with cpuload and few README changes.

Reply:
the request per second reported is completely useless... it looks like it is the requests per second since the start of the server.